src/Templates/NServiceBusWindowsService/Host.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+namespace NServiceBusWindowsService
+{
+    internal class Host
```

Comment:
Is there a reason to keep all the redundant `internal` and `private` modifiers around?

Reply:
I prefer to always have them, but @bording makes [a good case for removing them](https://github.com/Particular/dotnetTemplates/pull/4#discussion_r151274622), so I'm going to remove them.